src/functions/Output.ps1

```diff
@@ -494,11 +498,11 @@ function Get-WriteScreenPlugin ($Verbosity) {
 
         # if ($Context.AnyFocusedTests) {
         #     $focusedTests = $Context.FocusedTests
-        #     & $SafeCommands["Write-Host"] -ForegroundColor Magenta "There are some ($($focusedTests.Count)) focused tests '$($(foreach ($p in $focusedTests) { $p -join "." }) -join ",")' running just them."
+        #     & $SafeCommands["Write-Host"] -ForegroundColor $ReportTheme.TestRunBlock "There are some ($($focusedTests.Count)) focused tests '$($(foreach ($p in $focusedTests) { $p -join "." }) -join ",")' running just them."
```

Comment:
This is commented out and won't be used. Probably Find&Replace error?

Reply:
removed